docs/Reference/ApiKeys.md

```diff
@@ -551,4 +551,99 @@ const client = new jsc8({url: "https://gdn1.macrometa.io", token: "XXXX"});
 const client = new jsc8({url: "https://gdn1.macrometa.io", apiKey: "XXXX"});
 
 const result = await client.setBillingAccessLevel("some-key", "rw");
+```
+
+## client.listApikeyAttributes
+
+`async client.listApikeyAttributes(keyid)`
+
+List the attributes of a specific key-id for an apikey.
+
+**Arguments**
+
+- **keyid**: `string`
+    
+    The id of the api key.
+
+**Examples**
+
+```js
+const client = new jsc8({url: "https://gdn1.macrometa.io", token: "XXXX"});
+//---- OR ----
+const client = new jsc8({url: "https://gdn1.macrometa.io", apiKey: "XXXX"});
+
+const result = await client.listApikeyAttributes("some-key");
+```
+
+## client.createApikeyAttributes
+
+`async client.createApikeyAttributes(keyid, data)`
+
+Sets the attributes of a specific key-id for an apikey.
+
+**Arguments**
+
+- **keyid**: `string`
+    
+    The id of the api key.
+
+- **data**: `object`
+    
+    - key value pair to be set as attributes for the specific key-id for an apikey.
+
+**Examples**
+
+```js
+const client = new jsc8({url: "https://gdn1.macrometa.io", token: "XXXX"});
+//---- OR ----
+const client = new jsc8({url: "https://gdn1.macrometa.io", apiKey: "XXXX"});
+
+const result = await client.createApikeyAttributes("some-key", "{ key: value }");
+```
+## client.updateApikeyAttributes
+
+`async client.createApikeyAttributes(keyid, data)`
+
+Updates the attributes of a specific key-id for an apikey.
+
+**Arguments**
+
+- **keyid**: `string`
+    
+    The id of the api key.
+
+- **data**: `object`
+    
+    - key value pair to be updated as attributes for the specific key-id for an apikey.
```

Comment:
follow the same format. either add or remove the hyphen at the start of the line. just check what we follow.

Reply:
i followed the same from billing access level

Reply:
@VikasPatel resolved